build-system/server/routes/list.js

```diff
@@ -180,7 +180,7 @@ router.get('/ecommerce-nested-menu', function(req, res) {
     'menu': [
       {
         'title': 'Clothing, Shoes, Jewelry \u0026 Watches',
-        'image': 'https://cdn.orvis.com/images/082119_w_new.jpg',
+        'image': 'https://picsum.photos/id/535/367/267',
```

Comment:
In a future PR can we make these served by the same system providing the JSON?

Reply:
We have some sample images in `example/img`. We haven't been super consistent about downloading all of the images we use into that folder and serving from that folder. We often use picsum simply because they have high quality pretty photos. It could be worthwhile to download a set of UI-friendly jpgs, organize, and document them for testing purposes.

Reply:
Filed https://github.com/ampproject/amphtml/issues/25045 as a follow up to address in a future PR.